lib/compat/wordpress-5.8/block-editor.php

```diff
@@ -208,7 +208,7 @@ function gutenberg_get_default_block_editor_settings() {
 	}
 
 	$editor_settings = array(
-		'__unstableEnableFullSiteEditingBlocks' => gutenberg_supports_block_templates(),
+		'__unstableEnableFullSiteEditingBlocks' => current_theme_supports( 'block-templates' ),
```

Comment:
I noticed that this `__unstableEnableFullSiteEditingBlocks` flag is not set on Core at all, I'm not entirely sure what's the impact here but potentially we have a missing backport as well cc @noisysocks @gziolo

Reply:
Oh yes there was a ticket for that.

https://core.trac.wordpress.org/ticket/54583

Reply:
That's actually another flag.

Reply:
Ok I just saw that all the usage of `__unstableEnableFullSiteEditingBlocks` is actually inside `process.env.GUTENBERG_PHASE === 2` meaning it's not necessary at all to backport this to Core.